website/client/src/pages/settings/components/yourBalance.vue

```diff
@@ -11,6 +11,7 @@
       class="balance-info"
       :currency-needed="currencyNeeded"
       :amount-needed="amountNeeded"
+      :neededCurrencyOnly="true"
```

Comment:
This sets `neededCurrencyOnly` to `true` in all uses of the `yourBalance` component, not just in the `classSetting.vue` context. If that's what we want, we don't need to pass a property through at all, necessitating a rewrite of this PR; if we're only intending to change it in `classSetting`, this is a bug.